src/riot-rs-threads/src/lib.rs

```diff
@@ -17,6 +18,10 @@ pub mod thread_flags;
 
 pub use riot_rs_runqueue::{RunqueueId, ThreadId};
 pub use thread_flags as flags;
+// Used by `autostart_thread`
```

Comment:
(out of scope but still): is there a way to hide these pubs (here and in riot-rs)?
like, put them in a module that we hide from docs?
```Rust
#[hide-from-docs]
pub mod reexports {
  pub use ...;
}
```